app/services/bgs/dependent_service.rb

```diff
@@ -65,7 +66,7 @@ def submit_686c_form(claim)
         # why I am deliberately raising these errors here.
         validate_file_number_format!(file_number:)
         validate_file_number_matches_ssn!(file_number:)
-        BGS::SubmitForm686cJob.perform_async(uuid, claim.id, form_hash_686c)
+        BGS::SubmitForm686cJob.perform_async(uuid, @icn, claim.id, form_hash_686c)
```

Comment:
(Non blocking, also this is in the existing code so not a comment on your work) what is icn? It seems like acronyms are widely used throughout this codebase and I always wonder why we don't have clearer names for these things.

Reply:
My understanding is that it's an identifier for the Veteran. I passed it here as opposed to calling it within the jobs from my first pass at looking at maybe removing user dependencies completely. I figured I could keep it passed like this to remove one more dependency within the job.